api4/post.go

```diff
@@ -211,7 +211,7 @@ func getPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	} else {
 		w.Header().Set(model.HEADER_ETAG_SERVER, post.Etag())
-		w.Write([]byte(post.ToJson()))
+		w.Write([]byte(c.App.PostWithProxyAddedToImageURLs(post).ToJson()))
```

Comment:
Maybe we should have this function and the one for lists that is added everywhere be generic. So something like `DoPostRewriting` and have it call PostWithProxyAddedToImageURLs. So if we end up doing more rewriting it's not a pain to add everywhere.

Reply:
I wanted the APIs to explicitly rewrite the image URLs, to facilitate the API parameter we talked about before that would allow disable server-side rewriting and allow client-side URL rewriting.

Reply:
Alright. I guess we don't need to solve this problem now.